apps/expo/src/components/notif/Conversation.tsx

```diff
@@ -1,47 +1,62 @@
-import { TouchableOpacity, View, Text } from "react-native"
-import TimeAgo from "~/components/timeAgo/TimeAgo"
+import { Text, TouchableOpacity, View } from 'react-native'
+import { Link } from 'expo-router'
+
 import { MaterialCommunityIcons } from '@expo/vector-icons'
-import { useGlobalContext } from "~/context/global-context"
-import { Link } from "expo-router"
 import { ChatType } from '@prisma/client'
 
+import TimeAgo from '~/components/timeAgo/TimeAgo'
+import { useGlobalContext } from '~/context/global-context'
+import { api } from '~/utils/api'
+
 export interface ConversationProps {
-  chatId: number,
-  messageContent: string,
-  createdAt: Date,
-  chatName: string,
-  readBy: number[],
-  type: ChatType,
+  chatId: number
```

Comment:
Good practice for interfaces is to use semicolons to separate fields, for symmetry with class declarations:

https://ts.dev/style/#member-property-declarations

Reply:
This is how Prettier formatted it. If we want to change it then we need to change our Prettier configuration.